KmsClient.js

```diff
@@ -2,11 +2,18 @@
  * Copyright (c) 2019-2021 Digital Bazaar, Inc. All rights reserved.
  */
 import base64url from 'base64url-universal';
+import {cryptoLd} from './cryptoLd.js';
 import {httpClient, DEFAULT_HEADERS} from '@digitalbazaar/http-client';
 import {signCapabilityInvocation} from 'http-signature-zcap-invoke';
-import * as sec from 'security-context';
-import {cryptoLd} from './cryptoLd.js';
-const securityContextUrl = sec.constants.SECURITY_CONTEXT_V2_URL;
+import * as webkmsContext from 'webkms-context';
+
+const {CONTEXT_URL: WEBKMS_CONTEXT_URL} = webkmsContext;
+const symmetric = new Map([
+  ['AesKeyWrappingKey2019',
+    'https://w3id.org/security/suites/aes-2019/v1'],
+  ['Sha256HmacKey2019',
+    'https://w3id.org/security/suites/hmac-2019/v1']
```

Comment:
There are no context modules associated with these URL correct?

```suggestion
  [
    'AesKeyWrappingKey2019',
    'https://w3id.org/security/suites/aes-2019/v1'
  ], [
    'Sha256HmacKey2019',
    'https://w3id.org/security/suites/hmac-2019/v1'
  ]
```

Reply:
@mattcollier There are context modules for those, https://github.com/digitalbazaar/aes-key-wrapping-2019-context and https://github.com/digitalbazaar/sha256-hmac-key-2019-context. Should grab the context url from the modules, right?

Reply:
Yes, I think it would be best to use the URL exports from those contexts.

Reply:
addressed here 64db6e5